ipaserver/install/krainstance.py

```diff
@@ -134,8 +134,13 @@ def configure_instance(self, realm_name, host_name, dm_password,
 
             self.step("enabling KRA instance", self.__enable_instance)
 
+        if ra_only:
```

Comment:
Not sure if this is necessary. Is kra install ever executed with `ra_only=True` option?